Examples/UIExplorer/js/ListViewExample.js

```diff
@@ -62,9 +62,9 @@ var ListViewSimpleExample = React.createClass({
         noSpacer={true}
         noScroll={true}>
         <ListView
+          removeClippedSubviews={true}
```

Comment:
I agree, however it is already true by default if not specified (https://github.com/facebook/react-native/blob/master/Libraries/CustomComponents/ListView/ListView.js#L500). Up to you if you want to keep it to make it more explicit.